pages/zh/lb4/readmes / loopback-next / examples / todo/README.md

```diff
@@ -0,0 +1,96 @@
+# @loopback/example-todo
```

Comment:
I believe this page corresponds to https://loopback.io/doc/en/lb4/todo-tutorial.html. If that's the case, the file should be zh/lb4/todo-tutorial.md. 

To add some background, there are some README.md files in the loopback-next packages or extensions are being copied to loopback.io. Since we keep the LB4 translation in loopback.io, we can follow the corresponding md file for the translation files. For example, if we're translating https://loopback.io/doc/en/lb4/todo-tutorial.html,
- the english version: https://github.com/strongloop/loopback.io/blob/gh-pages/pages/en/lb4/todo-tutorial.md
- the translated version: https://github.com/strongloop/loopback.io/blob/gh-pages/pages/LANG/lb4/todo-tutorial.md

Reply:
Got it.